README.md

```diff
@@ -617,6 +617,8 @@ we provide a standard VagrantFile with the required components enabled. Simply r
 
 This should be sufficient to create a Kind cluster and run Tetragon.
 
+If you are getting an error, you can try to run `sudo launchctl load /Library/LaunchDaemons/org.virtualbox.startup.plist` (from https://stackoverflow.com/questions/18149546/macos-vagrant-up-failed-dev-vboxnetctl-no-such-file-or-directory).
+
```

Comment:
Instead of making this note twice, how about we add a link  to the vagrant section in development/README.md?

Reply:
how about the current version? you can preview on https://github.com/dmitris/tetragon/blob/vagrant-fix/README.md#btf-requirement

Reply:
I think we could also remove the text above that, and say "To run with [vagrant](...), please see the instructions in the [Development Guilde](...). WDYT?